### DISCUSSION TITLE
[Bug]: Storage metrics page shows "Invalid number" for some cameras

### DISCUSSION THREAD

**Discussion by leccelecce** [blakeblackshear/frigate#14576] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

`/api/recordings/storage` can return `usage: null` if there are no recordings, resulting in "Invalid number" being displayed in the UI.

![image](https://github.com/user-attachments/assets/e5675651-3609-4933-9e94-3dd07e23090e)

### Steps to reproduce

1. Have a camera with no recordings
2. View the system metrics page

### Version

0.15.0-4ff0c8a

### Frigate config file

```yaml
n/a
```

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

n/a

### Screenshots of the Frigate UI's System metrics pages

Provided

**Comment by hawkeye217** [comment, marked as answer]
Fixed in https://github.com/blakeblackshear/frigate/pull/14587

**Reply by leccelecce** [reply]
confirmed fixed, thanks!
